students/FlyStriker/hw-3/answer_2.md

```diff
@@ -0,0 +1,15 @@
+1 + 1   //number , два числа = число 
+'1' + 1 //string , если один из елементов строка то неявно приводиться к строке
+1 + [2] //number неявное привидение типа к числу 
+1 + {}  //number неявное привидение типа к числу несмотря на то что 2 елемент object 
```

Comment:
```suggestion
1 + {}  // String - '1[object Object]'. Because of {}.valueOf() is not a primitive, so toString fallback used, String(1) + String({}), '1' + '[object Object]', '1[object Object]'
```